bigquery/tests/unit/test_dataset.py

```diff
@@ -186,11 +186,22 @@ def test_from_string(self):
         self.assertEqual(got.project, "string-project")
         self.assertEqual(got.dataset_id, "string_dataset")
 
+    def test_from_string_w_prefix(self):
+        cls = self._get_target_class()
+        got = cls.from_string("prefix:string-project.string_dataset")
+        self.assertEqual(got.project, "string-project")
```

Comment:
Shouldn't this be `prefix:string-project`, since the prefix is actually part of the project ID?